packages/rainbowkit/src/components/AsyncImage/AsyncImage.tsx

```diff
@@ -62,11 +62,12 @@ export function AsyncImage({
               'src': src,
             }
           : {
-              backgroundSize: 'cover',
+              backgroundSize: 'contain',
```

Comment:
Since `EIP6963` connectors have their own icon it might look a bit over sized if i do `cover`. Just made sure that both rainbowkit connectors and `EIP6963` connectors have background size `contain` so it fills up the space better.